### ISSUE TITLE
login page shows blank

### ISSUE THREAD

**Issue by tmysoft** [OS4ED/openSIS-Classic#268]
Hi There, I installed the 8.0 and latest snapshot/main branch on Linux/Fedora server, I could finish all the setup, but when it redirected me to the login page, it shows nothing, just blank. I am using php8, could this be the reason that it does not work as expected?
Thanks!

**Comment by rogergrant99** [comment]
Indeed, using php 8.0 causes that problem, try using 7.4.30 , that's the one I am using and it's working fine.

**Comment by openSISAdmin** [comment]
We released a new version of openSIS and included server setup instructions within the docs older. Please download the latest version and try it out.

**Comment by king-hub259** [comment]
its only showing blank page during the process of installation

**Comment by fcortesjp** [comment]
like @rogergrant99 pointed out... php 7.4 is the one that seems to be playing nice with the 8/9.x opesis installs
with php 8.3 I get blank page so I switched back to 7.4 on my prod centos 7 install and on my centos 9 stream dev install

**Comment by sayan-os4ed** [comment]
Please use the updated code from the repo and check to see if you are still getting this error or not.

Thanks!
